faq.json

```
@@ -1,13 +1,13 @@
 {
-  "areyouabot": "Well, this is, however many of the other users in this room are not, as it is actually a Matrix room and only bridged to Discord. The bridged Matrix users will just appear to be bots on the Discord side.",
+  "areyouabot": "Well, this is a bot, however many of the other users in this room are not, as it is actually a Matrix room and only bridged to Discord. The bridged Matrix users will just appear to be bots on the Discord side.",
   "bestvpn": "There is no perfect VPN for every individual and their threat model. We recommend people use our VPN Chart to compare top services and contribute data to the open source project at https://techlore.tech/vpnchart.html. Current top recommendations with an emphasis on privacy include ProtonVPN, IVPN, Mullvad, & Windscribe.",
   "bots": "We have a Discord bridge, Mjolnir to help with moderation, a Plexus bot to quickly search for de-googled Android app compatibility, and this FAQ bot. If you have other bots that might be useful for this room, please contact Jonah or another admin.",
-  "chromiumvsfirefox": "Chromium is well-known for its high security, and has a much more restrictive sandbox and more mitigations than Firefox. Chromium forks like Brave or Ungoogled Chromium are among the best in terms of out of the box telemetry (and even Chrome is better than Firefox in this regard), and Brave also offers better tracking protection than any other browser without extensions. Firefox, on the other hand, while less secure, has some interesting privacy features, such as tab containers or cookie isolation. It also has a very high amount of configurability using about:config. For the Tor network, it is recommended to use Tor Browser (based on Firefox), as it has quite a bit of fingerprinting protection.",
+  "chromiumvsfirefox": "Chromium is well-known for its high security, and has a much more restrictive sandbox and more mitigations than Firefox. Chromium forks like Brave or Ungoogled Chromium are among the best in terms of out of the box telemetry (and even Chrome is better than Firefox in this regard), and Brave also offers better tracking protection than any other browser without extensions. Firefox, on the other hand, while less secure, has some interesting privacy and security features, such as HTTPS-only mode or per site cookie isolation. It also has a very high amount of configurability using about:config. For the Tor network, it is recommended to use Tor Browser (based on Firefox-ESR), as it has quite a bit of fingerprinting protection. Please note that Tor browser should be used on the `Safest` security level, as that grants the most security and privacy you can get out of it.",
```

Don't scare away new users.
```suggestion
  "chromiumvsfirefox": "Chromium is well-known for its high security, and has a much more restrictive sandbox and more mitigations than Firefox. Chromium forks like Brave or Ungoogled Chromium are among the best in terms of out of the box telemetry (and even Chrome is better than Firefox in this regard), and Brave also offers better tracking protection than any other browser without extensions. Firefox, on the other hand, while less secure, has some interesting privacy and security features, such as HTTPS-only mode or per site cookie isolation. It also has a very high amount of configurability using about:config.",
```

---

> Don't scare away new users.

You removed the pre-existing Tor part too. Also, recommending to use a browser on lowest security settings that is based on a rarely updated version of an already low security browser (as stated in the first part of this entry, "chromiumvsfirefox") wouldn't be a good advice that everyone could get behind, in my opinion. Your call, though.

---

Actually, the preexisting part can be removed, even if I didn't intend to do so. And I never recommended using Tor Browser on lowest security settings.

---

I see. Well, I assumed that is what you meant by "dont scare away new users", since the part where I said max settings stuff got removed. Alright, I guess a new entry about anonymity wouldn't hurt.